### PR TITLE
Feat/jest bdd

### PR DESCRIPTION
**Pull request by eoln**
typescript, jest, bdd toolset for PISP repo

### CODE REVIEW

**Comment by eoln** [comment]
> Cool.
> 
> We will want to remove some config later on such as unit tests and test coverage as the pisp repo has no notion of such things, but this is fine for now.

I would keep the setup for unit testing, 
yeah, coverage doesn't have any sense for this case... fixing...

File: .gitignore copy [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Logs
```
Comment (lewisdaly) [inline .gitignore copy:1]: Is this file needed?

Comment (eoln) [reply .gitignore copy]: uuups removed copy!
